acquire/acquire.py

```diff
@@ -1217,9 +1217,14 @@ class RemoteAccess(Module):
         ("glob", "sysvol/Program Files/TeamViewer/*.log"),
         ("glob", "sysvol/Program Files (x86)/TeamViewer/*.log"),
         ("glob", "AppData/Roaming/TeamViewer/*.log", from_user_home),
-        # anydesk
+        # anydesk - Windows
         ("dir", "sysvol/ProgramData/AnyDesk"),
         ("glob", "AppData/Roaming/AnyDesk/*.trace", from_user_home),
+        ("dir", "AppData/AnyDesk/", from_user_home),
+        ("dir", "AnyDesk/", from_user_home),
```

Comment:
Are these correct?

Reply:
Changed this (misunderstood the environment variables in the documentation) and changed from collecting entire directory to only the .trace files, such that we won't be surprised by the size of the other files that are present ;)